device/core/src/module_client.ts

```diff
@@ -222,4 +222,25 @@ export class ModuleClient extends EventEmitter {
   static fromAuthenticationProvider(authenticationProvider: AuthenticationProvider, transportCtor: any): ModuleClient {
     return InternalClient.fromAuthenticationProvider(authenticationProvider, transportCtor, ModuleClient) as ModuleClient;
   }
+
+  /**
+   * @description         Creates an IoT Hub device client by using configuration
+   *                      information from the environment. If an environment
+   *                      variable called `EdgeHubConnectionString` or `IotHubConnectionString`
```

Comment:
>EdgeHubConnectionString [](start = 43, length = 23)

Why would these env variables not follow the same naming conventions as the other? IOTEDGE_CONNECTION_STRING and IOTHUB_CONNECTION_STRING?

Reply:
Not sure. The other SDKs seem to follow this convention. Agree that it should be consistent though. We shipped public preview of Edge with these variable names and it has continued from there.

Reply:
ok i guess...

---
In reply to: [194575469](https://github.com/Azure/azure-iot-sdk-node/pull/310#discussion_r194575469) [](ancestors = 194575469)